### ISSUE TITLE
amplify init fails on my windows 10 machine

### ISSUE THREAD

**Issue by rg3h** [aws-amplify/amplify-cli#10889]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

node version: v16.16.0  npm version: 8.17.0

### Amplify CLI Version

9.2.1

### What operating system are you using?

Windows 10

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

no changes. Just followed the amplify tutorial:
https://docs.amplify.aws/start/getting-started/setup/q/integration/js/#initialize-a-new-backend

### Amplify Categories

Not applicable

### Amplify Commands

init

### Describe the bug

amplify init
reproducible windows 10 15Aug22
Pick the defaults. Here is a copy of the operations and first error:
C:\rcg\src\projects\amplify\amplify-js-app>amplify init
Note: It is recommended to run this command from the root of your app directory
? Enter a name for the project amplifyjsapp
The following configuration will be applied:

Project information
| Name: amplifyjsapp
| Environment: dev
| Default editor: Visual Studio Code
| App type: javascript
| Javascript framework: none
| Source Directory Path: src
| Distribution Directory Path: dist
| Build Command: npm.cmd run-script build
| Start Command: npm.cmd run-script start

? Initialize the project with the above configuration? Yes
Using default provider  awscloudformation
? Select the authentication method you want to use: AWS profile

For more information on AWS Profiles, see:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html

? Please choose the profile you want to use default
Error: ENOENT: no such file or directory, open 'C:\rcg\.aws\config'
    at Object.openSync (fs.js:497:3)
    at Object.openSync (pkg/prelude/bootstrap.js:739:32)
    at Object.readFileSync (fs.js:393:35)
    at Object.readFileSync (pkg/prelude/bootstrap.js:1025:36)
    at Object.readFileSync (C:\snapshot\repo\build\node_modules\aws-sdk\lib\util.js:95:26)
    at IniLoader.parseFile (C:\snapshot\repo\build\node_modules\aws-sdk\lib\shared-ini\ini-loader.js:6:47)
    at IniLoader.loadFrom (C:\snapshot\repo\build\node_modules\aws-sdk\lib\shared-ini\ini-loader.js:56:30)
    at getRegion (C:\snapshot\repo\build\node_modules\aws-sdk\lib\node_loader.js:118:32)
    at Config.region (C:\snapshot\repo\build\node_modules\aws-sdk\lib\node_loader.js:173:18)
    at Config.set (C:\snapshot\repo\build\node_modules\aws-sdk\lib\config.js:524:39)
    at Config.<anonymous> (C:\snapshot\repo\build\node_modules\aws-sdk\lib\config.js:359:12)
    at Config.each (C:\snapshot\repo\build\node_modules\aws-sdk\lib\util.js:512:32)
    at new Config (C:\snapshot\repo\build\node_modules\aws-sdk\lib\config.js:358:19)
    at Object.loadServiceClass (C:\snapshot\repo\build\node_modules\aws-sdk\lib\service.js:108:16)
    at Object.Service [as constructor] (C:\snapshot\repo\build\node_modules\aws-sdk\lib\service.js:47:29)
    at new <anonymous> (C:\snapshot\repo\build\node_modules\aws-sdk\lib\util.js:629:24)
    at getConfiguredAmplifyClient (C:\snapshot\repo\build\node_modules\amplify-provider-awscloudformation\lib\aws-utils\aws-amplify.js:42:14)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Object.init (C:\snapshot\repo\build\node_modules\amplify-provider-awscloudformation\lib\amplify-service-manager.js:51:25)
    at async Object.run (C:\snapshot\repo\build\node_modules\amplify-provider-awscloudformation\lib\initializer.js:135:9) {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: 'C:\\rcg\\.aws\\config'
}

I then did a mkdir for C:/rcg/.aws and got a similar error for the .aws/config
I then mad C:/rcg/.aws/config and got a new error:
C:\rcg\src\projects\amplify\amplify-js-app>amplify init
Note: It is recommended to run this command from the root of your app directory
? Enter a name for the project amplifyjsapp
The following configuration will be applied:

Project information
| Name: amplifyjsapp
| Environment: dev
| Default editor: Visual Studio Code
| App type: javascript
| Javascript framework: none
| Source Directory Path: src
| Distribution Directory Path: dist
| Build Command: npm.cmd run-script build
| Start Command: npm.cmd run-script start

? Initialize the project with the above configuration? Yes
Using default provider  awscloudformation
? Select the authentication method you want to use: AWS profile

For more information on AWS Profiles, see:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html

? Please choose the profile you want to use default
Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:617:3)
    at Object.readSync (pkg/prelude/bootstrap.js:890:32)
    at tryReadSync (fs.js:382:20)
    at Object.readFileSync (fs.js:419:19)
    at Object.readFileSync (pkg/prelude/bootstrap.js:1025:36)
    at Object.readFileSync (C:\snapshot\repo\build\node_modules\aws-sdk\lib\util.js:95:26)
    at IniLoader.parseFile (C:\snapshot\repo\build\node_modules\aws-sdk\lib\shared-ini\ini-loader.js:6:47)
    at IniLoader.loadFrom (C:\snapshot\repo\build\node_modules\aws-sdk\lib\shared-ini\ini-loader.js:56:30)
    at getRegion (C:\snapshot\repo\build\node_modules\aws-sdk\lib\node_loader.js:118:32)
    at Config.region (C:\snapshot\repo\build\node_modules\aws-sdk\lib\node_loader.js:173:18)
    at Config.set (C:\snapshot\repo\build\node_modules\aws-sdk\lib\config.js:524:39)
    at Config.<anonymous> (C:\snapshot\repo\build\node_modules\aws-sdk\lib\config.js:359:12)
    at Config.each (C:\snapshot\repo\build\node_modules\aws-sdk\lib\util.js:512:32)
    at new Config (C:\snapshot\repo\build\node_modules\aws-sdk\lib\config.js:358:19)
    at Object.loadServiceClass (C:\snapshot\repo\build\node_modules\aws-sdk\lib\service.js:108:16)
    at Object.Service [as constructor] (C:\snapshot\repo\build\node_modules\aws-sdk\lib\service.js:47:29)
    at new <anonymous> (C:\snapshot\repo\build\node_modules\aws-sdk\lib\util.js:629:24)
    at getConfiguredAmplifyClient (C:\snapshot\repo\build\node_modules\amplify-provider-awscloudformation\lib\aws-utils\aws-amplify.js:42:14)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Object.init (C:\snapshot\repo\build\node_modules\amplify-provider-awscloudformation\lib\amplify-service-manager.js:51:25)
    at async Object.run (C:\snapshot\repo\build\node_modules\amplify-provider-awscloudformation\lib\initializer.js:135:9) {
  errno: -4068,
  syscall: 'read',
  code: 'EISDIR'
}

### Expected behavior

It appears there is a C:/Users/rcg/.aws but the system is going to C:/rcg/.aws  (I do have a C:/rcg).

### Reproduction steps

Using a windows 10 machine with node v16.16.0 and npm  v8.17.0 follow the tutorial at:
https://docs.amplify.aws/start/getting-started/setup/q/integration/js/#initialize-a-new-backend

Once you have created the javasscript project type: amplify init and select the defaults

I was able to repeat this.

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Project Identifier

I could not run amplify diagnose --send-report  as it was failing before the project was created.

### Log output

<details>

```
# Put your logs below this line
These instructions assume you are on a linux machine. There is no ~/ on windows,


```

</details>

### Additional information

I suspect amplify init is using an environment variable or assuming something about the user's home directory?  We can look at what I have set up.  Windows uses things like:
APPDATA=C:\Users\rcg\AppData\Roaming
HOME=/cygdrive/c/rcg
HOMEDRIVE=C:
HOMEPATH=\Users\rcg
ProgramData=C:\ProgramData
PROGRAMFILES=C:\Program Files
ProgramFiles(x86)=C:\Program Files (x86)
SYSTEMDRIVE=C:
SYSTEMROOT=C:\WINDOWS
USERNAME=rcg
USERPROFILE=C:\Users\rcg
WINDIR=C:\WINDOWS

So I wonder if the script is using HOME rather than HOMEPATH or  USERPROFILE whereas setting up amplify does put  .aws and .amplify in C:/Users/rcg

They should probably use the same environment vars when determining where to look for .aws and .amplify

**Comment by ykethan** [comment]
hey @rg3h, to could you try using the `aws configure` command then paste in the credentials for the amplify user. 
Could you then try to perform `amplify init` in a new directory or project and let us know if this still throws the errors?

**Comment by ykethan** [comment]
Hello @rg3h, following up on this issue. Please do let us know if you require any assistance on this.

**Comment by rg3h** [comment]
Thanks, I ended up backing off amplify and using lightsail.
I'd like to try amplify again after I get my site up and running. Feel free
to close this ticket and when I do try again, I'll be sure to run amplify
init.  I suspect this error came from an early attempt to use amplify and
mixing and matching the GUI and the CLI incorrectly. I probably need an
"amplify clean-out-everything-and-try-again" command while learning.

thanks again,
rich

**Comment by ykethan** [comment]
Hey @rg3h, on a init failure if you would like clean up the project by deleting the amplify folder(if created), remove the aws-exports file in src(if created), check the amplify console if an application was created and delete it. 
Primarily, `amplify delete` command can be utilized to clean up resources.

As discussed I will close the issue, please do reach out to us again if you require any assistance.

**Comment by rg3h** [comment]
Great thanks. I think I had other residue like environment vars, maybe some
S3 resources? I poked around on AWS and removed things that looked like
they came from installing amplify (I messed up the first time by using the
GUI, then mid stream using CLI commands). Now that I am more familiar with
some things, I can probably take another pass at it.

I wonder if a sandbox would allow people to try out the features of amplify
without needing these vars?  Maybe that is hard to do given what needs to
be put on the local machine.

Thanks!
rich
